meshapp/src/profile/profile-page.tsx

```diff
@@ -0,0 +1,332 @@
+import React from "react";
+import {
+  Box,
+  Grid,
+  Typography,
+  createTheme,
+  ThemeProvider,
+  Chip,
+} from "@mui/material";
+
+import ProfileTextField from "../components/profile/profile-textfield";
+import { Profile } from "./types/profile";
+import "./styling/profile-page.css";
+
+const theme = createTheme({
```

Comment:
For theming like this it might make more sense to have this in a theme file or something like that so all pages can reuse these styles. Its not really a thing yet and there's not a task for it so this doesn't need to be done in this pr, but I just wanted to put that out there.